mvlearn/plotting/plot.py

```diff
@@ -135,10 +148,116 @@ def crossviews_plot(
 
     if title is not None:
         plt.tight_layout(rect=[0, 0.03, 1, 0.95])
-        plt.suptitle(title)
+        ax.set_title(title)
```

Comment:
Is this the same thing? before it was one title for all the plots and I recall it working all okay. Did something break?

Reply:
Oops, I didn't mean to change it for crossviews_plot, I meant to do it for quick_visualize. I'll change that back